Crashlytics/Crashlytics/Models/FIRCLSFileManager.m

```diff
@@ -124,6 +126,17 @@ - (BOOL)moveItemAtPath:(NSString *)path toDirectory:(NSString *)destDir {
   return YES;
 }
 
+- (BOOL)didCrashOnPreviousExecution {
+  NSString *crashedMarkerFileName = [NSString stringWithUTF8String:FIRCLSCrashedMarkerFileName];
+  NSString *crashedMarkerFileFullPath =
+      [[self rootPath] stringByAppendingPathComponent:crashedMarkerFileName];
+  static dispatch_once_t checkCrashFileMarketExistsOnceToken;
+  dispatch_once(&checkCrashFileMarketExistsOnceToken, ^{
```

Comment:
I would actually wrap the rest of this method (besides the return) so we don't pay the cost of file lookups more than once